dask/system.py

```diff
@@ -0,0 +1,57 @@
+import math
+import os
+import sys
+
+HAVE_PSUTIL = False
+
+try:
```

Comment:
Small style comment, we tend to do optional imports like:

https://github.com/dask/dask/blob/7b4f90f0a708cbb97ebb498c78f7931d82ec99db/dask/diagnostics/tests/test_profiler.py#L15-L18

and then have a `if psutil is not None:` check later when `psutil` is needed

Reply:
done, thanks!